CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing Guidelines
+
+Thank you for considering contributing to omniauth-mlh.
+
+## Contributor Checklist
+
+- Fork the Repo ( https://github.com/mlh/omniauth-mlh/fork )
+- Create your feature branch (`git checkout -b my-new-feature`)
+- Commit your changes (`git commit -am 'Add some feature'`)
+- Push to the branch (`git push origin my-new-feature`)
+- Create a new Pull Request
+
+## Local Testing Strategy
+
+To test omniauth-mlh locally, you can use [Sinatra application](https://gist.github.com/theycallmeswift/d8a8a22f95dd5d35f03661b19665e0d3). Here are the steps to set up and run the application:
+
+- Add files from [Sinatra application](https://gist.github.com/theycallmeswift/d8a8a22f95dd5d35f03661b19665e0d3) in a folder within a directory next to the checked out [omniauth-mlh](https://github.com/MLH/omniauth-mlh) library. 
```

Comment:
I'm inclined to say this should probably live in an `example` directory. But happy to defer to @theycallmeswift's take

Reply:
We could also generalize this the way this Gem does: https://github.com/auth0/omniauth-auth0/blob/master/CONTRIBUTING.md

Agree that we should avoid linking to a random gist